bedrock/firefox/templates/firefox/interest-dashboard.html

```diff
@@ -24,9 +24,6 @@
   {{ fxfamilynav('desktop', '', 'dark') }}
 {% endblock %}
 
-{% block tabzilla_css %}{% endblock %}
-{% block tabzilla_js %}{% endblock %}
-
```

Comment:
As noted above, I think we can change this block to `page_css` as long as `dashboard-base.html` gets updated as well.